components/04-reference-pages/07-faculty-profile-page/faculty-profile-page.hbs

```diff
@@ -102,6 +102,13 @@
       <div class="col-xxl-12 col-xl-12 col-lg-12 col-md-12 col-sm-12" id="results">
         <input type="hidden" value="000isg537" id="uuid" name="uuid" />
 
+        <input type="hidden" value="5" id="awardsCount" name="awardsCount" />
+        <input type="hidden" value="155" id="presentationsCount" name="presentationsCount" />
```

Comment:
should this one be 5 as well?

Reply:
I could change the value, but I was checking to ensure it would cut off at 5. The 155 checks to make sure if someone goes past the value, it will take the smaller value.